alerts/keys-api-outdated-keys.non-empty-db.rule.yml

```diff
@@ -0,0 +1,13 @@
+groups:
+  - name: Keys API. Update on non empty db
+    rules:
+      - alert: KeysApiOutdatedKeysNonEmptyDB
+        expr: |
+          (time() - lido_keys_api_last_update_timestamp{}) >= 10*60 and lido_keys_api_last_update_timestamp{} > 0
```

Comment:
Maybe add `time() - process_start_time_seconds{}) < 15 * 60` here?

Reply:
we discussed , that here 15 it is enough